docs/0.5/tutorials/configuring_splinter_nodes_with_docker.md

```diff
@@ -429,17 +429,27 @@ beta nodes.
     application authorization handler which handles the circuit’s change
     proposals.
 
+    `--service-peer-group gsBB,gsAA` Service peer group (a list of peer
+    services). Peer services are services used by peer nodes within a circuit.
+    This is the group of services that must come to consensus amongst the node
+    peers.
+
     `--service-arg *::admin_keys=$(cat /config/keys/beta.pub)`: String that
     passes key/value arguments to the specified service. As with
     `--service-type` above, the glob operator, `*`, can be used to match all or
     certain parts of the service ID. The `admin_keys` value specifies the public
     key of the smart contract administrator who's is allowed to upload smart
     contracts to the circuit.
 
-    `--service-peer-group gsBB,gsAA` Service peer group (a list of peer
-    services). Peer services are services used by peer nodes within a circuit.
-    This is the group of services that must come to consensus amongst the node
-    peers.
+    ***
```

Comment:
This just creates an <hr> - is there no way to create a separate box? Something akin to the note box you see here: https://docs.readthedocs.io/en/stable/config-file/index.html

Reply:
We were no formatting blockquotes before. I added a commit for that and upated the note to be in a blockquote.